compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -474,8 +474,8 @@ fn codegen_fn_body(fx: &mut FunctionCx<'_, '_, '_>, start_block: Block) {
                     *destination,
                 );
             }
-            TerminatorKind::UnwindTerminate => {
-                codegen_panic_cannot_unwind(fx, source_info);
+            TerminatorKind::UnwindTerminate(reason) => {
+                codegen_unwind_terminate(fx, source_info, *reason);
```

Comment:
@bjorn3 I think this is completely unreachable since cranelift doesn't seem to support unwinding. But the match arm already existed so I patched it as appropriate.